### PR TITLE
Bump haproxy package versions

### PR DESCRIPTION
**Pull request by hoerup**
https://redmine.pfsense.org/issues/9521

### CODE REVIEW

**Comment by rbgarga** [comment]
@PiBa-NL could you please take a look at it?

**Comment by PiBa-NL** [comment]
TLDR: this PR can be committed imho.

I think updating to 1.8 for the stable haproxy package and 2.0 for the haproxy-devel package is a good step to do. 1.8 has been stable for quite a while so no one should really be needing 1.7.x anymore, and 2.0.7 seems to be evolving to a pretty stable version as well, that wasn't quite so for the earlier 2.0.x releases.. (1.9 is a more 'short term' development release cycle, just as 2.1 will be.. so those should imho be skipped as a pfSense package..)

And yes it isn't really the latest/greatest haproxy-devel 2.1-devX port development branche as some people might think given the name of the pfSense package.. but to be honest that version gets updated to often to follow here and while offering 'bleeding edge' features they do also suffer from a variety of bugs that would need debugging. As such i believe that people wanting to use such a development version should be willing to compile their own version of haproxy binaries which then also them to bisect commits that caused a issue and applying/validating possible fixes to the the code that could be proposed by haproxy developers.

**Comment by hoerup** [comment]
I agree with @PiBa-NL that this is the best scenario for now. 
But depending on how haproxy 2.0 evolves and stabilizes during pfsense 2.5 development cycle, we might want to re-evaluate if a bump to haproxy 2.0 for pkg-pfsense-haproxy is reasonable.

File: net/pfSense-pkg-haproxy-devel/Makefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # $FreeBSD$
 
 PORTNAME=	pfSense-pkg-haproxy-devel
-PORTVERSION=	0.59
-PORTREVISION=	20
+PORTVERSION=	0.60
+PORTREVISION=	1
```
Comment (rbgarga) [inline net/pfSense-pkg-haproxy-devel/Makefile]: PORTREVISION line must be deleted when PORTVERSION changes

File: net/pfSense-pkg-haproxy/Makefile [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # $FreeBSD$
 
 PORTNAME=	pfSense-pkg-haproxy
-PORTVERSION=	0.59
-PORTREVISION=	19
+PORTVERSION=	0.60
+PORTREVISION=	1
```
Comment (rbgarga) [inline net/pfSense-pkg-haproxy/Makefile]: PORTREVISION line must be deleted when PORTVERSION changes

File: net/pfSense-pkg-haproxy-devel/Makefile (hunk 1 above)
Comment (hoerup) [reply net/pfSense-pkg-haproxy-devel/Makefile]: Done

File: net/pfSense-pkg-haproxy/Makefile (hunk 2 above)
Comment (hoerup) [reply net/pfSense-pkg-haproxy/Makefile]: Done
